packages/codebytes/package.json

```diff
@@ -27,7 +27,7 @@
     "@emotion/react": "^11.4.0",
     "@emotion/styled": "^11.3.0",
     "@loadable/component": "^5.13.1",
-    "react-monaco-editor": "0.34.0",
+    "@monaco-editor/react": "4.3.1",
     "react-resize-observer": "1.1.1",
     "monaco-editor": "0.20.0"
```

Comment:
since we are working with the latest version of `@monaco-editor/react` it may be worth updating the `monaco-editor` version as well

https://github.com/suren-atoyan/monaco-react/blob/cd924806506646e2270ade2df896b3603f1c3f30/package.json#L36

Reply:
awesome, thank you for the heads up!

Reply:
@aionate0812 The amazing thing is that I wasn't getting syntax highlighting before, but with the package updates, I am 🎉